crates/fluvio-service/src/server.rs

```diff
@@ -275,8 +208,6 @@ mod test {
         let reply2 = socket.send(&msg2).await.expect("send");
         trace!("received 2nd reply from server: {:#?}", reply2);
         assert_eq!(reply2.response.msg, "hello2");
-
-        shutdown.notify();
```

Comment:
why this is removed?

Reply:
I think I deleted this by mistake, good catch